resources/views/modal.blade.php

```diff
@@ -33,13 +33,13 @@ class="modal-content-wrapper md:m-auto w-full {{ $class }} {{ $widthClass }}"
         @if($style) style="{{ $style }}" @endif
     >
         <div class="modal-content dropdown-scrolling {{ $widthClass }}">
-            <div class="p-6 sm:p-12">
+            <div class="p-8 sm:p-10">
```

Comment:
won't you have less space now that you increased it to `p-8` on screens below `sm`? 🤔